README.md

```diff
@@ -151,7 +193,50 @@ then configure key protect service with authenticator. then run operations on se
             response = testService.actionOnKey(unWrapKeyOptionsModel).execute();
             responseObj = response.getResult();
 ```
-### Creating/Retrieving an Import token
+
+### Rotate a key
+```
+            InputStream inputstream = new FileInputStream(<PATH_TO_Rotate_Key_Body>)
+            ActionOnKeyOptions actionOnKeyOptionsModel = new ActionOnKeyOptions.Builder()
+                    .id(keyId)
+                    .bluemixInstance(bluemixInstance)
+                    .action("rotate")
+                    .keyActionOneOf(inputstream)
+                    .prefer("return=representation")
+                    .build();
+            Response<KeyActionOneOfResponse> response = testService.actionOnKey(actionOnKeyOptionsModel).execute();
+            KeyActionOneOfResponse responseObj = response.getResult();
+```
+
+### Disable a key
+```
+            InputStream inputstream = new FileInputStream(<PATH_TO_Disable_Key_Body>)
+            ActionOnKeyOptions disableKeyOptionsModel = new ActionOnKeyOptions.Builder()
+                    .id(keyId)
+                    .bluemixInstance(bluemixInstance)
+                    .action("disable")
+                    .keyActionOneOf(inputstream)
```

Comment:
I don't think there should be a key body for disable...?

Reply:
Agreed. Fixed code to make keyActionOneOf nullable.